code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -251,6 +251,48 @@ datum
 				..()
 				return
 
+		harmful/hemotoxin
+			name = "hemotoxin"
+			id = "hemotoxin"
+			description = "A toxin that causes tissue damage and bleeding."
+			reagent_state = LIQUID
+			fluid_r = 195
+			fluid_g = 120
+			fluid_b = 130
+			transparency = 130
+			depletion_rate = 0.6
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if (!M) M = holder.my_atom
+				M.take_toxin_damage(0.5 * mult)
+				take_bleeding_damage(M, null, rand(5,10) * mult, DAMAGE_STAB)
+				if (prob(33))
+					M.emote(pick("groan", "pale", "scream"))
+				if (prob(25))
+					boutput(M, pick("<span class='alert'>You feel like your flesh is melting!</span>", "<span class='alert'>You feel a sharp pain in your chest!</span>"))
+
+				..()
+				return
+
+		harmful/hemotoxinplus
```

Comment:
not a fan of adding a chemical with no way to obtain, seems useless

Reply:
i can remove it then

Reply:
Maybe make it possible from artbeakers?

Reply:
oops, i kinda deleted it...